api/src/opentrons/protocols/advanced_control/transfers.py

```diff
@@ -843,6 +861,20 @@ def _map_volume(i):
 
         return [_map_volume(i) for i in range(total)]
 
+    def _check_valid_volume_parameters(self, disposal_volume, air_gap, working_volume):
```

Comment:
Let's give type annotations to this new function.

```suggestion
    def _check_valid_volume_parameters(self, disposal_volume: float, air_gap: float, working_volume: float) -> None:
```

Normally, forgetting the type annotations would be an error because of [our project config](https://github.com/Opentrons/opentrons/blob/c0543a6e0a89e38a06117c5124085a4bb45443c4/api/mypy.ini#L5), but I think we have that overridden for this file because it's legacy code and we haven't gotten around to fixing it up. So we *could* leave them out. But let's leave a little less work for our future selves and add them now.